### PR TITLE
xcode-test-bundle-builder-for-bitrise 0.0.1

### PR DESCRIPTION
**Pull request by otusweb**
This is my first step, so feel free to point out things that need to be updated

### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

**Comment by otusweb** [comment]
@fadookie, figured you might be interested in this step

**Comment by viktorbenei** [comment]
Thanks @otusweb - we'll do a review ASAP! ;)

**Comment by viktorbenei** [comment]
I'd suggest you to drop the for Bitrise from the end of the ID, to make it shorter for us who work in Bitrise  YML a lot 😉

File: steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml [hunk 1]
```diff
@@ -0,0 +1,73 @@
+title: Build test bundle for AWS Device Farm
```
Comment (viktorbenei) [inline steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml]: Drop the aws part from the title, it should work with any other tool / service, right?

File: steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+summary: Build your XCTest or XCUITest bundle to upload to AWS Device Farm
```
Comment (viktorbenei) [inline steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml]: I'd also just say "for example AWS device far" as it should work with other services too

File: steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+description: Build your XCTest or XCUITest bundle to upload to AWS Device Farm
```
Comment (viktorbenei) [inline steps/xcode-test-bundle-builder-for-bitrise/0.0.1/step.yml]: Same thing here 😉

**Comment by viktorbenei** [comment]
If you don't want to change the ID we can accept the step as it is, otherwise:

1. just do the changes in your step's repo
2. create a new release there
3. share that new version

**Comment by fadookie** [comment]
Very cool! I'd agree with viktor that this step seems to be generic (which is good!) So no need to say it's explicitly for device farm.

**Comment by fadookie** [comment]
Also I left a few line notes on your repo.

**Comment by otusweb** [comment]
Closing will submit another pull request with requested change

**Pull request closed without merging**